x-pack/solutions/observability/plugins/investigate_app/scripts/evaluate/README.md

```diff
@@ -0,0 +1,55 @@
+# Investigation RCA Evaluation Framework
+
+## Overview
+
+This tool is developed for our team working on the Elastic Observability platform, specifically focusing on evaluating the Investigation RCA AI Integration. It simplifies scripting and evaluating various scenarios with the Large Language Model (LLM) integration.
+
+## Setup requirements
+
+- An Elasticsearch instance configured with cross cluster search pointing to the edge-rca cluster
+- A Kibana instance
+- At least one .gen-ai connector set up
+
+## Running evaluations
+
+### Prerequists
+
+#### Elasticsearch instance
+
+You'll need an instance configured with cross cluster search for the [edge-rca](https://studious-disco-k66oojq.pages.github.io/edge-rca/) cluster. To create one, utilize [oblt-cli](https://studious-disco-k66oojq.pages.github.io/user-guide/cluster-create-ccs/) and select `edge-rca` as the remote cluster.
+
+Once your cluster is created, paste the the yml config provided in your `kibana.dev.yml` file.
+
+#### Fixture data
+
+To load the fixtures needed for the tests, first run:
+
+`$ node x-pack/solutions/observability/plugins/investigate_app/scripts/load/index.js`
+
+### Executing tests
+
+Run the tool using:
+
+`$ node x-pack/solutions/observability/plugins/investigate_app/scripts/evaluate/index.js`
+
+This will evaluate all existing scenarios, and write the evaluation results to the terminal.
+
+### Configuration
+
+#### Kibana and Elasticsearch
+
+By default, the tool will look for a Kibana instance running locally (at `http://localhost:5601`, which is the default address for running Kibana in development mode). It will also attempt to read the Kibana config file for the Elasticsearch address & credentials. If you want to override these settings, use `--kibana` and `--es`. Only basic auth is supported, e.g. `--kibana http://username:password@localhost:5601`. If you want to use a specific space, use `--spaceId`
+
+For use with your configured oblt-cli cluster, utilize `--kibana http://admin:[YOUR_CLUSTER_PASSWORD]@localhost:5601`
```

Comment:
@dominiqueclarke Regarding the discussion we had about the error I got:

```
Error: Could not connect to Kibana: Expected HTTP 200 from http://kibana_system_user:PASSWORD@localhost:5601/, got 403
```

The solution is mentioned right here, so all good. My recommendation would be to either reorder the configuration part if that makes sense or even simpler to add a sentence after line 28 that points to the Kibana and Elasticsearch configuration

Reply:
Something like 

```
Before loading the fixtures, make sure to read the [Kibana and Elasticsearch configuration](#kibana-and-elasticsearch) section. To load the fixtures needed for the tests, first run:
```

Reply:
For now I updated the issue description to note the configuration part as well.